storage-service/src/config.ts

```diff
@@ -5,22 +5,43 @@ interface MinioConfig {
   port: number;
   bucketName: string;
 }
+interface AzureBlobConfig {
+  azureAccountName: string;
+  azureStorageUrl: string;
+  azureConnectionString: string | undefined;
+  containerName: string;
+}
 interface Config {
   port: number;
   allowOrigin: string;
+  storageProvider: string;
   storage: MinioConfig;
+  azureBlobStorage: AzureBlobConfig;
 }
 
 const config: Config = {
   port: Number(process.env.STORAGE_SERVICE_PORT) || 8090,
   allowOrigin: process.env.ACCESS_CONTROL_ALLOW_ORIGIN || "*",
+  storageProvider:
+    process.env.STORAGE_PROVIDER === "azure-storage"
```

Comment:
Maybe just `azure` would be sufficient?

Reply:
@SamuelPull since azure is quite generic I wanted to be a little more specific. It is also the name of docker container running locally (since azure will be too generic for container) and I wanted it to be the same. Are you also ok with "azure-storage" or do you prefer to change it to "azure"?

Reply:
it's okay, thanks for the explanation